.github/workflows/build-make-check.yml

```diff
@@ -0,0 +1,28 @@
+name: "Build with warnings and make check"
```

Comment:
Can we use something smaller? Like simple "Build"? This is quite long and redundant message in the commit status: `Build with warnings and make check / makecheck`, it can be just `Build / make`.

Reply:
The file itself can also be renamed to `build.yml`